free-programming-books-tr.md

```diff
@@ -139,7 +139,7 @@
 
 * [AB2014 Ruby Programlama Dili](https://github.com/leylaKapi/AB2014-Ruby-Programlama-Dili/blob/master/Ruby_AB2014.md)
 * [Ruby](https://www.ruby-lang.org/tr/)
-* [Ruby 101](https://www.gitbook.io/book/vigo/ruby-101)
+* [Ruby 101](https://vigo.gitbooks.io/ruby-101/content/)
```

Comment:
Yep. I don't expect gitbook to change their URL structure anytime soon. But if and when they do it, they'll most probably modify `/details` or `/content` and not the "base" URL leading to the book.

This redirect is perfectly fine for me. If you think this is not the right decision, I'd be glad to hear your point of view! :)